doc/build-unix.md

```diff
@@ -129,7 +129,7 @@ libqrencode (optional) can be installed with:
 
     sudo apt-get install libqrencode-dev
 
-Once these are installed, they will be found by configure and a bitcoin-qt executable will be
+Once these are installed, they will be found by configure and a Bitcoin-Qt executable will be
```

Comment:
the executable name was correct before

Reply:
The documentation uses many different variants. Is it entirely system dependent? In build-unix.md, for instance, it shows up in upper and lower case.

Reply:
It could be upper case on macOS, but I don't see a reason to change it here